src/V3LinkDot.cpp

```diff
@@ -158,6 +158,7 @@ class LinkDotState final {
     std::array<ScopeAliasMap, SAMN__MAX> m_scopeAliasMap;  // Map of <lhs,rhs> aliases
     std::vector<VSymEnt*> m_ifaceVarSyms;  // List of AstIfaceRefDType's to be imported
     IfaceModSyms m_ifaceModSyms;  // List of AstIface+Symbols to be processed
+    int m_stepNumber;  // Number of the step
     bool m_forPrimary;  // First link
     bool m_forPrearray;  // Compress cell__[array] refs
     bool m_forScopeCreation;  // Remove VarXRefs for V3Scope
```

Comment:
```suggestion
    const VLinkDotStep m_step;  // Operational step
```

Then can remove m_forPrimary etc and make those functions instead that test m_stepNumber.